shared/loadgenerator/loadgenerator.go

```diff
@@ -194,19 +194,35 @@ func (gr *GeneratorResults) SaveJSON() error {
 }
 
 // ErrorsPercentage returns the error percentage of the result[idx] based on response codes.
-// Any non 200 response will provide a value > 0.0
+// Any non 200 response will be counted as errors.
 func (gr *GeneratorResults) ErrorsPercentage(idx int) float64 {
```

Comment:
This is a breaking change as a few methods in serving that use this and this will break the auto-updater.

Reply:
It will not break the auto-updater. Either the function name or interface is not changed. I just added one extra function `ErrorsPercentageOverall`.

Reply:
yeah. Dont think this will break it.